tokio/tests/rt_common.rs

```diff
@@ -418,4 +418,67 @@ rt_test! {
         assert_eq!(buf, b"hello");
         tx.send(()).unwrap();
     }
+
+    mod local_set {
```

Comment:
What was the reason for nesting this?

Reply:
The intention was to be able to run just the local set tasks with `cargo test -- local_set`, but to still use the macro that generates tests for both the basic scheduler and the threaded scheduler.